index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
Before you push/submit your code, make sure to delete any empty spaces.

Reply:
Thanks Sebastian!